common/logging/tests/test-feature-test_logger/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[workspace]
```

Comment:
Nesting workspaces is considered bad practice, and leads to duplication/maintenance burden in the form of a second `Cargo.lock`.

I would be happy to continue without tests for the `test_logger` feature, because this quickly gets into the weeds of Cargo's build system for minimal benefit. See e.g. https://github.com/rust-lang/cargo/issues/2911 for background. As devs using Lighthouse everyday we will notice if we do something to break the `test_logger`, and a breakage would not be severe (some extra log output for tests only).

Alternatively, you might be able to wire the `test-feature-test_logger` crate into the top-level Cargo.toml to avoid the workspace duplication. If that works, I'm also happy to keep the test.

Reply:
Ok cool, that looks good. I was overthinking it trying to avoid any `cd {} && cargo test`, but I guess that's exactly what we want to test and hopefully it won't be too fragile :crossed_fingers: